### PR TITLE
Set flash_attn mask to None in LlamaPretrainedModelAuto

### PR DESCRIPTION
**Pull request by From00**
### PR types
Others

### PR changes
Others

### Description

Llama预训练模型通过`is_casual_mask`函数判断flash_attn的`attention_mask`参数，这在组网中引入了控制流操作，导致静半新IR无法跑通（新IR控制流机制正在支持中...）。
由于预训练场景下并不需要`attention_mask`，相关判断逻辑是冗余的，本PR临时修改组网，直接将`attention_mask`设置为`None`，以在组网中消除控制流操作。

### CODE REVIEW

**Review by ZHUI — APPROVED** [review]
LGTM

**Review by haohongxiang — APPROVED** [review]
LGTM
